packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -316,6 +316,7 @@ const PersonalDetails = ({
                                                                 }}
                                                                 {...field}
                                                                 required
+                                                                hide_disabled={false}
```

Comment:
Can we call this prop `hide_disabled_countries` for better clarification?

Reply:
Thanks, @akmal-binary. changed to should_hide_disabled_options as @aaron-binary  suggested